src/index.html

```diff
@@ -5,8 +5,405 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="head">
+      <a href="index.html" class="head__logo">
+        <img src="images/logo.png" alt="The logo of Moyo">
+      </a>
+
+      <nav class="head__nav">
```

Comment:
Why can't the navigation be a separate block?
Navigation is a separate block that can be independent of the header, navigation is also often used in the footer of the page, so I recommend moving navigation to a separate block and accordingly in a separate file.